pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/prepare-air-gapped/index.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: layout.pug
+navigationTitle: Prepare Air-Gapped Cluster
+title: Prepare Air-Gapped Cluster
+menuWeight: 10
```

Comment:
This should probably either be 5 or 15 
whether it should be before or after the "Prepare Local Environment" topic in the nav.

```suggestion
menuWeight: 15
```

Reply:
Totes, that makes more sense to have it pinned after the local env instructions.